db/migrate/20170327081658_add_price_to_products.rb

```diff
@@ -0,0 +1,5 @@
+class AddPriceToProducts < ActiveRecord::Migration[5.0]
+  def change
+    add_column :products, :price, :float, default: 0.0
```

Comment:
let's talk about what this column should be called. there will be different types of prices